### PR TITLE
Create back button

### LINKED ISSUE #30: Add a back button to app
As a user, I should be able to click a back button to go back to the original playlist/device selection screen from an actively started PH.

Additionally, the playback should pause when I click back.

### PR DESCRIPTION
**Pull request by ColemanMitch**
image

- [x] pause playback and return to playlist select screen
- [x] cache the active device into the device drop down  

Closes #30

### CODE REVIEW

**Comment by ColemanMitch** [comment]
One annoying little quirk that I haven't been able to quite figure out is how to get the app name center aligned once the party has been started. Check the screenshot below and note how `Currently Playing:`  and `Spowerfy 🍺` are not aligned... 🤔 I think it has to do with the flex-basis for the back button

image

**Comment by ColemanMitch** [comment]
@dwilliams27 have a look at this PR whenever you have a hot sec and lmk what you think

File: src/App.tsx [hunk 1]
```diff
@@ -46,6 +47,7 @@ class App extends Component<void, AppState> {
     this.pauseCurrentPlayback = this.pauseCurrentPlayback.bind(this);
     this.resumeCurrentPlayback = this.resumeCurrentPlayback.bind(this);
     this.loadDevices = this.loadDevices.bind(this);
+    this.goBack = this.goBack.bind(this);
```
Comment (dwilliams27) [inline src/App.tsx:52]: Random note, I _think_ in newer versions of React there's a way to set this up without all the annoying binding? I'll have to look into it

File: src/App.tsx [hunk 2]
```diff
@@ -188,12 +191,20 @@ class App extends Component<void, AppState> {
...
+    })
+  }
+
   render() {
     return (
       <AppContainer className="App">
         { this.state.partyStarted ?
           <PartyTime className="app-body">
             <header>
+            { this.state.activeSong ? <ArrowBackIos onClick={() => this.goBack()} style={{ float: 'left', cursor: 'pointer', color: 'white', marginTop: "1rem", marginLeft: "1rem"}}/> : "hi"}
```
Comment (dwilliams27) [inline src/App.tsx:216]: Lol may want a better fallback than "hi"

**Review by dwilliams27 — APPROVED** [review]
Looks good, one small comment

File: src/App.tsx (hunk 1 above)
Comment (ColemanMitch) [reply src/App.tsx:52]: you talking about function components (aka react hooks) or smth different?

File: src/App.tsx (hunk 2 above)
Comment (ColemanMitch) [reply src/App.tsx:216]: Lol, I have commited to this branch in like 6 mos. so I can definitely clean it up a bit before I merge into `main`. I have a merge conflict I need to resolve anyway...

**Pull request merged**